nab/detectors/htmjava/src/main/java/nab/detectors/htmjava/HTMModel.java

```diff
@@ -138,9 +144,13 @@ public Parameters getSpatialPoolerParams(JsonNode modelParams) {
         if (spParams.has("synPermActiveInc")) {
             p.set(KEY.SYN_PERM_ACTIVE_INC, spParams.get("synPermActiveInc").asDouble());
         }
-        if (spParams.has("seed")) {
-            p.set(KEY.SEED, spParams.get("seed").asInt());
-        }
+        // if (spParams.has("seed")) {
+        //     p.set(KEY.SEED, spParams.get("seed").asInt());
+        // }
+        // For some seeds, htm.java throws
+        // java.lang.ArrayIndexOutOfBoundsException: 1017
+        //    at org.numenta.nupic.Connections.computeActivity(Connections.java:1199)
+        p.set(KEY.SEED, 42);
```

Comment:
I think it would be better to open an Issue in HTM.Java, than making this comment a permanent feature inside this class. I don't think there's any reason to document bugs. They should just be fixed. IMHO